src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -18,44 +18,11 @@ freshnessValidatedDate: never
 
 New Relic's Kubernetes plugin for log forwarding simplifies sending logs from your cluster to New Relic logs. It uses a standalone Docker image and runs as a DaemonSet, seamlessly collecting logs for centralized analysis and troubleshooting. Forwarding your Kubernetes logs to New Relic will give you enhanced <InlinePopover type="logs" /> capabilities to collect, process, explore, query, and alert on your log data.
```

Comment:
@nbaenam We say there's a Kuberentes plugin for log forwarding, but we also say there's a Fluent Bit output plugin. It seems like these are two different names for the same thing, and is "plugin" the best term? Don't Kubenerntes plugins modify the Kubernetes cluster in some way? This Flurent Bit container is just forwarding logs, right?

I wonder if we should just drop "plugin" from the title and switch to something like this: "Forward Kubernetes log." If you did that, you'd have to rearrange the introduction and the Fluent Bit section.

Also, this doc repeats that confusing sentence: "New Relic has a [Fluent Bit output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin."

Again, aren't we just saying you can have two DaemonSets in your cluster?

* One DaemonSet for the Kubernetes integration
* Another DaemonSet for the Fluent Bit Docker container

Reply:
Hey @mangulonr, can you help us and clarify this a bit, please?  Thanks!